### DISCUSSION TITLE
Secrets on self-hosted QFieldCloud

### DISCUSSION THREAD

**Discussion by SolangeD** [opengisch/QField#4771] [Q&A]
Hi all,

We are installing QFieldCloud on our own server. Our goal is to have a connection to a postgis layer.
I saw that [here ](https://docs.qfield.org/reference/qfieldcloud/secrets/) that QFieldCloud deals with a postgis connection using secrets. 
When connecting to my admin interface (port 8011), I see no way to add secrets. The interface is indeed different from the one shown in the screenshot in the qfieldcloud documentation:
![image](https://github.com/opengisch/QField/assets/23310170/61824a41-3387-4f10-a026-6da703163cd7)

I am therefore wondering if there is a way to make a secure connection to a postgis layer on a self hosted QFieldCloud (I'd like to avoid writing the connection details in clear in the QGIS project file).

**Comment by suricactus** [comment]
Secrets configuration is currently unavailble from Django Admin. It is in our backlog list, but no immediate plans to be integrated. If you are interested to speed up development of the Secrets UI in Django Admin, drop us an email at sales@qfield.cloud .

**Reply by Baptenda** [reply]
Can you guide me how to access this in the backend so I can add the service file?

**Comment by boardend** [comment]
Note that secrets can now be added/modified through the admin panel with QFieldCloud version >= 0.27.0
